### ISSUE TITLE
New submissions for Thu, 12 May 22

### ISSUE THREAD

**Issue by imjunwei** [imjunwei/ocr-daily-paper#195]
### Arbitrary Shape Text Detection via Boundary Transformer
 - **Authors:** Shi-Xue Zhang, Xiaobin Zhu, Chun Yang, Xu-Cheng Yin
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2205.05320
 - **Pdf link:** https://arxiv.org/pdf/2205.05320
 - **Abstract**
 Arbitrary shape text detection is a challenging task due to its complexity and variety, e.g, various scales, random rotations, and curve shapes. In this paper, we propose an arbitrary shape text detector with a boundary transformer, which can accurately and directly locate text boundaries without any post-processing. Our method mainly consists of a boundary proposal module and an iteratively optimized boundary transformer module. The boundary proposal module consisting of multi-layer dilated convolutions will compute important prior information (including classification map, distance field, and direction field) for generating coarse boundary proposals meanwhile guiding the optimization of boundary transformer. The boundary transformer module adopts an encoder-decoder structure, in which the encoder is constructed by multi-layer transformer blocks with residual connection while the decoder is a simple multi-layer perceptron network (MLP). Under the guidance of prior information, the boundary transformer module will gradually refine the coarse boundary proposals via boundary deformation in an iterative manner. Furthermore, we propose a novel boundary energy loss (BEL) which introduces an energy minimization constraint and an energy monotonically decreasing constraint for every boundary optimization step. Extensive experiments on publicly available and challenging datasets demonstrate the state-of-the-art performance and promising efficiency of our method.
## Keyword: Text Recognition
There is no result 
### Pre-trained Language Models as Re-Annotators
 - **Authors:** Chang Shu
 - **Subjects:** Computation and Language (cs.CL); Artificial Intelligence (cs.AI); Human-Computer Interaction (cs.HC); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2205.05368
 - **Pdf link:** https://arxiv.org/pdf/2205.05368
 - **Abstract**
 Annotation noise is widespread in datasets, but manually revising a flawed corpus is time-consuming and error-prone. Hence, given the prior knowledge in Pre-trained Language Models and the expected uniformity across all annotations, we attempt to reduce annotation noise in the corpus through two tasks automatically: (1) Annotation Inconsistency Detection that indicates the credibility of annotations, and (2) Annotation Error Correction that rectifies the abnormal annotations. We investigate how to acquire semantic sensitive annotation representations from Pre-trained Language Models, expecting to embed the examples with identical annotations to the mutually adjacent positions even without fine-tuning. We proposed a novel credibility score to reveal the likelihood of annotation inconsistencies based on the neighbouring consistency. Then, we fine-tune the Pre-trained Language Models based classifier with cross-validation for annotation correction. The annotation corrector is further elaborated with two approaches: (1) soft labelling by Kernel Density Estimation and (2) a novel distant-peer contrastive loss. We study the re-annotation in relation extraction and create a new manually revised dataset, Re-DocRED, for evaluating document-level re-annotation. The proposed credibility scores show promising agreement with human revisions, achieving a Binary F1 of 93.4 and 72.5 in detecting inconsistencies on TACRED and DocRED respectively. Moreover, the neighbour-aware classifiers based on distant-peer contrastive learning and uncertain labels achieve Macro F1 up to 66.2 and 57.8 in correcting annotations on TACRED and DocRED respectively. These improvements are not merely theoretical: Rather, automatically denoised training sets demonstrate up to 3.6% performance improvement for state-of-the-art relation extraction models.
### Who won? Winner Determination and Robustness in Liquid Democracy
 - **Authors:** Matthias Bentert, Niclas Boehmer, Maciej Rymar, Henri Tannenberg
 - **Subjects:** Computer Science and Game Theory (cs.GT)
 - **Arxiv link:** https://arxiv.org/abs/2205.05482
 - **Pdf link:** https://arxiv.org/pdf/2205.05482
 - **Abstract**
 Liquid democracy is a decision-making paradigm in which each agent can either vote directly for some alternative or (transitively) delegate its vote to another agent. To mitigate the issue of delegation cycles or the concentration of power, delegating agents might be allowed to specify multiple delegation options. Then, a (cycle-free) delegation is selected in which each delegating agent has exactly one representative. We study the winner determination problem for this setting, i.e., whether we can select a delegation such that a given alternative wins (or does not win). Moreover, we study the robustness of winning alternatives in two ways: First, we consider whether we can make a limited number of changes to the preferences cast by the delegating or directly voting agents such that a given alternative becomes a winner in one/in all delegations, and second, whether we can make a limited number of changes to a selected delegation to make a given alternative a winner.
## Keyword: Handwriting
There is no result 
## Keyword: Scene Text
There is no result 
## Keyword: Text Segmentation
There is no result 
## Keyword: TextSpotter
There is no result 
## Keyword: Text Spotting
There is no result
